### PR TITLE
Assignment -7

### PR DESCRIPTION
**Pull request by ohunazz**

### CODE REVIEW

File: assignment/assignment-7/index.html [hunk 1]
```diff
@@ -5,87 +5,151 @@
...
+            <div class="feedback">
+                <label for="feedbackInput">Satisfaction Rating</label>
+                <input
+                    type="range"
+                    name="satisfactionRating"
+                    id="feedbackInput"
+                    min="1"
+                    max="99"
+                />
+                <label for="commentsInput">Comments</label>
```
Comment (ghost) [inline assignment/assignment-7/index.html:106]: commentsInput should be commentsTextarea.

File: assignment/assignment-7/index.html [hunk 2]
```diff
@@ -5,87 +5,151 @@
...
+                </button>
+            </div>
+            <div class="clickable-img">
+                <label for="imgInput">Icon Image</label>
+                <input
+                    type="image"
+                    name="iconImg"
+                    id="imgInput"
+                    src="./image.png"
+                />
```
Comment (ghost) [inline assignment/assignment-7/index.html:152]: If you can add autofocus that would be goog.

File: assignment/assignment-7/index.html (hunk 1 above)
Comment (ohunazz) [reply assignment/assignment-7/index.html:106]: good idea

File: assignment/assignment-7/index.html (hunk 2 above)
Comment (ohunazz) [reply assignment/assignment-7/index.html:152]: I added. Thanks

**Pull request closed without merging**